docker-compose.yml

```diff
@@ -1,4 +1,3 @@
-version: "3.9"
```

Comment:
Pourquoi l'enlever ?

Reply:
Docker affiche un warning à chaque lancement qui dit que cette propriété est obsolète et qu'il faut la supprimer 

```WARN[0000] /Users/pascal/Documents/dev/DataForGood/13_brigade_coupes_rases/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion
```

Reply:
arf, j'ai peut-etre une version obsolete sur mon ordi... c'est bizarre, normalement je pensait que locker une version serait une bonne chose... :thinking: